Index: .idea/Moodle-Bodio.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"WEB_MODULE\" version=\"4\">\r\n  <component name=\"FacetManager\">\r\n    <facet type=\"Python\" name=\"Python facet\">\r\n      <configuration sdkName=\"Python 3.13 (Moodle-Bodio)\" />\r\n    </facet>\r\n  </component>\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.tmp\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/temp\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/tmp\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/venv\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n    <orderEntry type=\"library\" name=\"Python 3.13 (Moodle-Bodio) interpreter library\" level=\"application\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Moodle-Bodio.iml b/.idea/Moodle-Bodio.iml
--- a/.idea/Moodle-Bodio.iml	(revision 1e8fffe29a425dc60a1db4f6b2f1d57f23298985)
+++ b/.idea/Moodle-Bodio.iml	(date 1743672517717)
@@ -2,7 +2,7 @@
 <module type="WEB_MODULE" version="4">
   <component name="FacetManager">
     <facet type="Python" name="Python facet">
-      <configuration sdkName="Python 3.13 (Moodle-Bodio)" />
+      <configuration sdkName="Python 3.10 (Moodle-Bodio)" />
     </facet>
   </component>
   <component name="NewModuleRootManager">
@@ -14,6 +14,6 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Python 3.13 (Moodle-Bodio) interpreter library" level="application" />
+    <orderEntry type="library" name="Python 3.10 (Moodle-Bodio) interpreter library" level="application" />
   </component>
 </module>
\ No newline at end of file
